pinot-common/src/main/java/com/linkedin/pinot/common/config/IndexingConfig.java

```diff
@@ -113,6 +116,15 @@ public void setSortedColumn(List<String> sortedColumn) {
     _sortedColumn = sortedColumn;
   }
 
+  
+  public List<String> getBloomFilterColumns() {
```

Comment:
Did not want to overload the config. Its possible we need bloom filter without inverted index and vice versa. Also sorted index does not need an entry in invertedIndexColumns but we might need bloom filter for a sorted column

Reply:
Can we add bloomfilter for all dimension columns? 

I am wondering if the configuration can be simplified - as it stands, we have one more tunable; if in the common case the overhead is much lower compared to the benefit, I would much rather have it on by default. We can have a global config to enable or disable bloom - so that way we can do a controlled rollout.

Also, I am guessing generating bloom will be handled in the offline segment creation path (pbnj) too by default? (We have overrides that disable inverted-index creation, so checking what the stance will with this feature)

Reply:
I agree with the concern that this is another config. Unfortunately creating bloom filter on all dimensions will add to memory usage (at least in the given implementation where the bitset is on heap) and additional overhead. The right solution would be to have a component that looks at the workload and tunes it automatically.

Currently, bloom filter is created when the segment is getting loaded. We can add the optimization to create it during segment generation in another PR.